app/javascript/util/languageUtil.tsx

```diff
@@ -164,14 +164,18 @@ export const getSfGovUrl = (enLink: string, node?: number, path?: string) => {
  */
 export function renderMarkup(translatedString: string, allowedTags?: string) {
```

Comment:
I grepped for the regex `renderMarkup\([^t]` and found that this function is used for free text fields in some places.